src/components/look-controls.js

```diff
@@ -227,34 +227,27 @@ module.exports.Component = registerComponent('look-controls', {
   /**
    * Handle positional tracking.
    */
-  updatePosition: (function () {
-    var deltaHMDPosition = new THREE.Vector3();
-
-    return function () {
-      var el = this.el;
-      var currentHMDPosition;
-      var currentPosition;
-      var position = this.position;
-      var previousHMDPosition = this.previousHMDPosition;
-      var sceneEl = this.el.sceneEl;
-
-      if (!sceneEl.is('vr-mode')) { return; }
+  updatePosition: function () {
+    var el = this.el;
+    var currentHMDPosition;
+    var currentPosition;
+    var position = this.position;
+    var previousHMDPosition = this.previousHMDPosition;
```

Comment:
Is previousHMDPosition defined if not vr-mode? Move below return?

Reply:
Probably harmless, we have an informal code style to define vars up top and alias `this.` variables to remove `this.` clutter.